src/docs/cookbook/persistence/sqlite.md

```diff
@@ -312,10 +312,15 @@ To run the example:
 ```dart
 import 'dart:async';
 
+import 'package:flutter/widgets.dart';
+
 import 'package:path/path.dart';
 import 'package:sqflite/sqflite.dart';
 
 void main() async {
+  // Avoid errors coused by flutter upgrade. 
```

Comment:
@basashi-G, can you correct this line?

  // Avoid errors coused by flutter upgrade.  =>   // Avoid errors caused by flutter upgrade.